src/v0.15/guide/index.md

```diff
@@ -190,5 +190,5 @@ upfront "wiring" rather than imperative handlers spread throughout your
 codebase.
 
 Although Orbit does not pretend to absorb all the complexity of writing
-ambitious data-driven applications, it's hoped that Orbit's composeable and
+ambitious data-driven applications, it's hoped that Orbit's composable and
```

Comment:
/shrug I'm not sure which spelling of this is correct but line 176 uses "composable" so this at least makes them consistent

Reply:
good catch on the inconsistency - "composable" works for me 👍